x/archivedb/key.go

```diff
@@ -0,0 +1,88 @@
+// Copyright (C) 2019-2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package archivedb
+
+import (
+	"encoding/binary"
+	"errors"
+	"math"
+)
+
+var (
+	ErrInsufficientLength = errors.New("packer has insufficient length for input")
+	longLen               = 8
```

Comment:
could we use `wrappers.LongLen`?

additionally, could we use `const` to define the constants and leave `var` where we cannot use `const`?